targets/runtimeDependencies/project.json

```diff
@@ -3,7 +3,9 @@
     "win7-x86": {},
     "win7-x64": {},
     "osx.10.10-x64": {},
-    "ubuntu.14.04-x64": {}
+    "ubuntu.14.04-x64": {},
+    "ubuntu.16.04-x64": {},
+    "ubuntu.16.10-x64": {}
```

Comment:
16.10 is in the build tools project.json so I figured I'd add it here.  I technically tested this on 16.04.  Should I not include 16.10?